client/src/components/SectionTable/SectionTable.js

```diff
@@ -78,7 +79,25 @@ class SectionTable extends PureComponent {
                     ) : (
                         <Fragment />
                     )}
+
+                    <a
+                        target="blank"
+                        style={{
+                            marginTop: '0px',
+                            fontSize: '15px',
+                        }}
+                        href={
+                            'https://zotistics.com/?&selectQuarter=&selectYear=&selectDep=' +
+                            urlEncode +
+                            '&classNum=' +
+                            courseDetails.courseNumber +
+                            '&code=&submit=Submit'
+                        }
+                    >
+                        Zotistics
+                    </a>
```

Comment:
_Side Note:_ In the future, we want to avoid  using `style` inside of components. Instead we give the component a class name and define the css at the top of the file. However since we're just copying the styling from prereq, it's fine for now. We'll come back later and clean it up in #122